### PR TITLE
Add script for running an ecs task and awaiting the result

### PR DESCRIPTION
**Pull request by meerkat-b**
We will need this when we eventually want to run a docker container containing our integration tests in the ECS cluster. this script gives us a mechanism to be able to pass/fail based on the result when using circieci

### CODE REVIEW

**Review by andrea-medeghini-ft — APPROVED** [review]

File: run_task_and_await_result.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+function wait_for_result { 
+    while [[ "$TASK_STATUS" != "STOPPED" ]]; do 
+        sleep 3
+        TASK_JSON=$(aws ecs describe-tasks --cluster $CLUSTER --tasks $TASK_ARN)
+        TASK_STATUS=$(jq -r .tasks[0].containers[0].lastStatus <<< $TASK_JSON)
+        echo "Current task status is $TASK_STATUS"
+    done
+
+    TASK_EXIT_CODE=$(jq -r .tasks[0].containers[0].exitCode <<< $TASK_JSON)
+    echo "Task staus eventually reached $TASK_STATUS"
```
Comment (tsaastamft) [inline run_task_and_await_result.sh]: Typo: "staus"

File: run_task_and_await_result.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+wait_for_result
+
+echo "Exit code for container $(jq .tasks[0].containers[0].name <<< $TASK_JSON) was $TASK_EXIT_CODE"
```
Comment (tsaastamft) [inline run_task_and_await_result.sh]: Hmm slightly confusing that `TASK_JSON` is only assigned to within that function. Maybe set it to empty first then `global TASK_JSON` (I think the syntax was) from inside the function. But NBD :)

Comment (meerkat-b) [reply run_task_and_await_result.sh]: fair enough, i'm all about that clarity

File: run_task_and_await_result.sh (hunk 1 above)
Comment (meerkat-b) [reply run_task_and_await_result.sh]: thanks

File: run_task_and_await_result.sh (hunk 2 above)
Comment (meerkat-b) [reply run_task_and_await_result.sh]: maybe i'll extract the status refresh to a separate function all together

Comment (meerkat-b) [reply run_task_and_await_result.sh]: Actually, my bash isn't eloquent away to think of a better way to make it more readable. I think it's not too bad as it is but if it really becomes a problem then I can  refactor a little

**Review by tsaastamft — APPROVED** [review]
This good, me approve

**Pull request merged**